src/MfaVerifyController.js

```diff
@@ -127,6 +127,15 @@ function (Okta, Checkbox, BaseLoginController, CookieUtil, TOTPForm, YubikeyForm
         }
       }
       return false;
+    },
+
+    back: function() {
+      // Empty function on verify controllers to prevent users
+      // from navigating back during 'verify' using the browser's
+      // back button. The URL will still change, but the view will not
+      // We will need to think of a better solution for this, possibly
```

Comment:
No strong opinion. Slight preference to creating a jira that has this info, and adding the jira number here instead of the "We will need to think of ..."